discord/utils/__init__.py

```diff
@@ -71,63 +65,3 @@
     "MISSING",
     "UNICODE_EMOJIS",
 )
-
-
-async def get_or_fetch(obj, attr: str, id: int, *, default: Any = MISSING) -> Any:
```

Comment:
Since this is removed, can you make sure it's not used anywhere else, and open an issue about this so we can work on it once the cache stuff is done ?